### PR TITLE
validating thumbprint

### PR DESCRIPTION
**Pull request by KrishnaAdityaB**

### CODE REVIEW

**Comment by RoopeshNair** [comment]
Looks good to me. Just ensure that we trim trailing spaces if any

**Comment by mvvsubbu** [comment]
@KrishnaAdityaB  why did the PR closed without merging?

File: Extensions/IISWebAppDeploy/Tests/Tasks/IISWebAppMgmt/ManageIISWebApp.Tests.ps1 [hunk 1]
```diff
@@ -152,6 +156,55 @@ Describe "Tests for testing Validate-Inputs functionality" {
             { Validate-Inputs -createWebsite "false" -websiteName " " -createAppPool "false" -appPoolName " " } | Should Not Throw
         }
     }
+
```
Comment (kmkumaran) [inline Extensions/IISWebAppDeploy/Tests/Tasks/IISWebAppMgmt/ManageIISWebApp.Tests.ps1]: Can you add a test for trimming whitespace in the end.

Comment (KrishnaAdityaB) [inline Extensions/IISWebAppDeploy/Tests/Tasks/IISWebAppMgmt/ManageIISWebApp.Tests.ps1]: It's already added. see above for the change in this file
